core/state/statedb.go

```diff
@@ -300,6 +300,13 @@ func (self *StateDB) AddBalance(addr common.Address, amount *big.Int) {
 	}
 }
 
+func (self *StateDB) SubBalance(addr common.Address, amount *big.Int) {
```

Comment:
Please add a method doc (would also appreciate one for others working with balances that are not covered). I know it's a bit out of scope for this PR, but I think it would be nice to consciously try to fix our code quality issues, even if they go out of way a bit from time to time :)